library/include/updater.h

```diff
@@ -48,10 +48,11 @@ extern "C" {
 /**
  * Configures updater.  First parameter is a struct containing configuration
  * from the running app.  Second parameter is a YAML string containing
- * configuration compiled into the app.
+ * configuration compiled into the app.  Returns true on success and false on
+ * failure. If false is returned, the updater library will not be usable.
```

Comment:
Should we include what should be done in the case it returns false? Would re-running `shorebird_init` be advised? Is there any way to recover?